apps/network-alerts/test/unit/notification/notification_test.js

```diff
@@ -1,15 +1,11 @@
 /* global MocksHelper,
   NotificationHandler,
-  Utils
  */
 'use strict';
 
-require('/test/unit/mock_utils.js');
 require('/js/notification/notification.js');
 
-var mocksForNotification = new MocksHelper([
-  'Utils'
-]);
+var mocksForNotification = new MocksHelper([]);
 
 suite('Network Alerts - Notification handling', function() {
   mocksForNotification.attachTestHelpers();
```

Comment:
heh :) The rule of thumb for me is if removing redundant code doesn't break existing tests and everything is green - I go ahead :)